### DISCUSSION TITLE
How to config

### DISCUSSION THREAD

**Discussion by amathar** [gui-dos/DiaBLE#8] [Q&A]
As I see, the is a TestFlight, fine.
What do I have to do, after installing DiaBLE via TestFlight when using a Libre 3?

**Comment by gui-dos** [comment]
amathar, welcome aboard!

Can you scan your Libre 3 successfully via NFC (the top right button)?

I am not activating the sensor or configuring it this way, just accessing its extended `patchInfo` which is now of 24 bytes instead of 6 bytes like the previous generation.

In the Console some internal details of the Libre 3 should be displayed, for example its activation time, its regional code, the current BLE PIN and MAC address, the internal status (typically `04` for an active sensor), ... it is just the beginning of a long journey!

If you switch to the third "Test" mode in the Console you should be aBLE to sniff the incoming data from the official app (or any other third-party app) already running in parallel.

Currently DiaBLE is unaBLE to decrypt those data, but if you login into your LibreLinkUp account from the second "Online" tab you should be aBLE to get the current glucose curve and the trend arrow, since "Trident" (that's their nickname for the official app) is uploading the data regularly every minute.

Differently than other obscure "WebFollower" solutions, I am connecting directly to the libreview.io servers and logging all the data traffic.

One more thing... The Apple Watch implementation is sharing most of the code with the iOS version but it is completely independent B-)
